code/modules/cargo/console.dm

```diff
@@ -77,7 +77,8 @@
 		data["supplies"][P.group]["packs"] += list(list(
 			"name" = P.name,
 			"cost" = P.cost,
-			"id" = pack
+			"id" = pack,
+			"desc" = P.desc ? P.desc : P.name // If there is a description, use it. Otherwise use the pack's name.

```

Comment:
replace this with `"desc" = P.desc || P.name` for a more concise version of the same behavior